dnf/cli/commands/module.py

```diff
@@ -43,15 +44,16 @@ def run_on_module(self):
             mods = self.base.repo_module_dict
 
             if self.opts.all:
-                print(mods.get_brief_description_all(self.opts.module_nsvp))
+                logger.info(mods.get_brief_description_all(self.opts.module_nsvp))
```

Comment:
you should not log output of list or info command.

Reply:
No, many times that's the first thing I check when resolving problem.